gobblin-kubernetes/gobblin-service/basic-cluster/application.yaml

```diff
@@ -0,0 +1,80 @@
+# In the future, build the kubernetes cluster from the official docker account
+# Also ensure that proper tagging/versioning is done i.e. remove :latest tag and instead use the digest of the container
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: 'gaas-app'
+  labels:
+    app: gaas-app
+spec:
+  selector:
+    matchLabels:
+      app: gaas-app
+  replicas: 1
+  template:
+    metadata:
+      name: 'gaas-app'
+      labels:
+        app: gaas-app
+    spec:
+      volumes:
+        - name: 'shared-jobs'
+          persistentVolumeClaim:
+            claimName: shared-jobs-claim
+        - name: 'shared-template-catalogs'
+          persistentVolumeClaim:
+            claimName: shared-template-catalogs-claim
+      containers:
+        - name: gobblin-service
+          image: will97/gobblin-as-a-service:latest
+          volumeMounts:
+            - name: shared-jobs
+              mountPath: /tmp/gobblin-service/jobs
+            - name: shared-template-catalogs
+              mountPath: /tmp/templateCatalog
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: 'gobblin-standalone-app'
+  labels:
+    app: gobblin-standalone-app
+spec:
+  selector:
+    matchLabels:
+      app: gobblin-standalone-app
+  replicas: 1
+  template:
+    metadata:
+      name: 'gobblin-standalone-app'
+      labels:
+        app: gobblin-standalone-app
+    spec:
+      volumes:
+        - name: 'shared-jobs'
+          persistentVolumeClaim:
+            claimName: shared-jobs-claim
+      containers:
+
+        - name: gobblin-standalone
+          image: will97/gobblin-standalone:latest
+          volumeMounts:
+            - name: shared-jobs
+              mountPath: /tmp/gobblin-standalone/jobs
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: gaas-svc
+  labels:
+    app: gobblin-service
+spec:
+  type: NodePort
```

Comment:
`Ingress` and `NodePort` are 2 different way to expose a Service to the outside. If we have one, we don't need the other. Checkout https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types (search for string "You can also use Ingress to expose your Service")

Reply:
Okay I'll change the service type to a clusterIP